test/sample_consensus/test_sample_consensus_quadric_models.cpp

```diff
@@ -387,9 +387,9 @@ TEST (SampleConsensusModelCylinder, RANSAC)
   Eigen::VectorXf coeff;
   sac.getModelCoefficients (coeff);
   EXPECT_EQ (7, coeff.size ());
-  EXPECT_NEAR (-0.5, coeff[0], 1e-3);
-  EXPECT_NEAR ( 1.7, coeff[1], 1e-3);
-  EXPECT_NEAR ( 0.5, coeff[6], 1e-3);
+  EXPECT_NEAR (-0.49, coeff[0], 1e-3);
```

Comment:
I guess here the author of the unit test generated a cylinder point cloud with `-0.5`, `1.7`, and`0.5` parameters, so they are "ground truth" expected values. I'd rather relax the near-ness threshold.